introspection.go

```diff
@@ -92,8 +92,9 @@ type ExchangeRuntimeState struct {
 
 // PeerRuntimeState is the runtime state for a single peer.
 type PeerRuntimeState struct {
-	HostPort    string                   `json:"hostPort"`
-	Connections []ConnectionRuntimeState `json:"connections"`
+	HostPort            string                   `json:"hostPort"`
+	OutboundConnections []ConnectionRuntimeState `json:"outbound connections"`
```

Comment:
json tag is the field name in the json struct, so you shouldn't have spaces.
use
```go
`json:"outboundConnections"`
`json:"inboundConnections"`
```